### ISSUE TITLE
Can't explode Korean language args in Windows10 pmmp

### ISSUE THREAD

**Issue by wsj7178** [pmmp/PocketMine-MP#2266]
### Issue description

In Windows, open the pmmp server and enter the command in Korean, which is delivered with spaces in the args variable at onCommand.

https://github.com/pmmp/PocketMine-MP/blob/05af87e1d4dfe99195715e23009477e906af8b36/src/pocketmine/command/SimpleCommandMap.php#L247
It does not completely separate Korean in Windows 10.

- Expected result: What were you expecting to happen?
- Actual result: What actually happened?

### Steps to reproduce the issue

1. ... Open pmmp server on windows10
2. ... Type the command in Korean.

### OS and versions

* PocketMine-MP: PocketMine-MP 4.0.0+dev.1214
* PHP: 7.2
* Server OS: Windows 10
* Game version: PE & Win10

### Plugins
- This issue can only be identified with a plugin

### Crashdump, backtrace or other files
- Do not paste crashdumps into an issue - please use our Crash Archive at https://crash.pmmp.io for submitting crash reports to not spam the issue tracker. Add links to your reports in the Crash Archive here.
- Please use gist or anything else to add other files and add links here

* This issue does not have any Crashdump, backtrace or other files

**Comment by solo5star** [comment]
![](https://i.imgur.com/hJArdi2.png)

here are some results. Each commands are `/땅 정보`, `/시세 확인`, `/워프생성 테스트 테스트워프`, `givemoney solo 12000`

tested with this code
```php
public function onCommandPreprocess(PlayerCommandPreprocessEvent $event){
	$commandLine = $event->getMessage();
	$args = array_map("stripslashes", str_getcsv($commandLine, " "));

	$this->getServer()->getLogger()->info("Message : " . $commandLine);
	var_dump($args);
}
```

**Comment by dktapps** [comment]
why are you using development builds >.<

I can only assume that this is some character encoding issue. Maybe related to #2228 .

**Comment by wsj7178** [comment]
If I use implode() instead, the problem is resolved.

https://github.com/organization/KoreanCommand/blob/master/src/koreancommand/command/KoreanCommandMap.php#L20

**Comment by dktapps** [comment]
hmm... so it would seem that this is an issue with `str_getcsv()`.

**Comment by dktapps** [comment]
This will be fixed in the 3.5.12 and 3.6.2 releases, and whatever dev builds arrive later today.
